crates/router/src/routes/dummy_connector.rs

```diff
@@ -4,24 +4,70 @@ use router_env::{instrument, tracing};
 use super::app;
 use crate::services::{api, authentication as auth};
 
+mod consts;
+mod core;
 mod errors;
 mod types;
 mod utils;
 
+#[instrument(skip_all, fields(flow = ?types::Flow::DummyPaymentCreate))]
+pub async fn dummy_connector_authorize_payment(
+    state: web::Data<app::AppState>,
+    req: actix_web::HttpRequest,
+    path: web::Path<String>,
+) -> impl actix_web::Responder {
+    let flow = types::Flow::DummyPaymentAuthorize;
+    let attempt_id = path.into_inner();
+    let payload = types::DummyConnectorPaymentConfirmRequest { attempt_id };
+    api::server_wrap(
+        flow,
+        state.get_ref(),
+        &req,
+        payload,
+        |state, _, req| core::payment_authorize(state, req),
+        &auth::NoAuth,
```

Comment:
I had this thought regarding the security of these routes. Can we have a lightweight authentication based on redis? what do you think? A little bit of protection with something similar to token based protection will help a lot.

Reply:
But we are showing expired page right. If we have an authentication here, we can't show that.